database/migrations/2016_04_04_193247_create_newApps_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateNewAppsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('newApps', function (Blueprint $table) {
```

Comment:
**Code Quality - Critical**
@andela-araimi The table name here is not explanatory enough, change it to some better and and also change the column name to be simpler